test/common/vmmanage.py

```diff
@@ -126,6 +140,12 @@ def build_and_install(install_image, build_image, args):
                 build_and_maybe_install(build_image, do_install=False, skips=skips, args=args)
             if install_image:
                 only_install(install_image, skips, args=args)
+
+            # Atomics need a companion image for tests
+            if build_image and install_image and install_image in testinfra.ATOMIC_IMAGES:
+                skips.append("cockpit-ostree")
```

Comment:
I don't quite understand why we don't want to install cockpit-ostree on Atomic systems...